client/ayon_core/hosts/houdini/plugins/load/load_hda.py

```diff
@@ -3,10 +3,28 @@
 from ayon_core.pipeline import (
     load,
     get_representation_path,
+    AVALON_CONTAINER_ID
 )
-from ayon_core.hosts.houdini.api import pipeline
+from ayon_core.hosts.houdini.api import lib, pipeline
+import hou
 
 
+def get_avalon_container():
+    path = pipeline.AVALON_CONTAINERS
+    avalon_container = hou.node(path)
+    if not avalon_container:
+        # Let's create avalon container secretly
+        # but make sure the pipeline still is built the
+        # way we anticipate it was built, asserting it.
+        assert path == "/obj/AVALON_CONTAINERS"
+
+        parent = hou.node("/obj")
```

Comment:
if we can support publishing from non object level, maybe it is also good idea to support digital asset loading in non-object network>

Reply:
> maybe it is also good idea to support digital asset loading in non-object network?

I had this thought too and also loading in the current opened network.
But, currently, artists expects every single loaded product to exist inside avalon container subnetwork.

Reply:
Maybe we can discuss later if there are some user needs, since HDA is mostly a package of the nodes to be used in external package.